tensorflow_model_analysis/eval_saved_model/load.py

```diff
@@ -134,7 +134,11 @@ def _load_and_parse_graph(self):
         else:
           raise ValueError('unrecognised suffix for metric. key was: %s' % k)
 
```

Comment:
Can we add a test for this in eval_saved_model/integration_test.py? You can make use of the fixed_prediction_estimator in the test to ensure you have a known expected loss value.